docs/adding-trusted-committers.md

```diff
@@ -0,0 +1,14 @@
+This document describes how to add a new Trusted Committer to the InnerSource Learning Path.
+
+1. Share your suggestion for new Trusted Commiter in the [:lock:learning-path-trusted-committers][learning-path-trusted-committers] private _Slack_ channel.
+2. If people seem to agree with your suggestion, then move forward.
+3. Create a pull request to the [Trusted Committers] section of the README with the new person.
+4. Invite the person into the [#learning-path-core] and [:lock:learning-path-trusted-committers][learning-path-trusted-committers] _Slack_ channels.
```

Comment:
> #learning-path-core

Are we making use of this, or does the private TCs channel supersede it?

Reply:
I think we could delete the “-core” version.

Reply:
> Does it really need the closed version?

This is potentially a bit sensitive. On the one hand, 'rejecting' a candidate TC, or having some other difficult discussion publicly, might not be good for psychological safety...but maybe having those discussions in the open will force people to be constructive with feedback e.g. "You're not quite ready for TC - here's what we'd like to see before granting you TC".